lib/membrane/rtp/payload_type_decoder.ex

```diff
@@ -0,0 +1,40 @@
+defmodule Membrane.RTP.PayloadTypeDecoder do
```

Comment:
It should be a part/submodule of RTP.Packet. Or at least it shouldn't end with Decoder